blocks/api/raw-handling/test/integration/index.js

```diff
@@ -23,8 +24,7 @@ describe( 'raw handling: integration', () => {
 	beforeAll( () => {
 		// Load all hooks that modify blocks
 		require( 'blocks/hooks' );
-		// Load all blocks
-		require( 'blocks/library' );
+		registerAllBlocks();
```

Comment:
Is this all blocks or just the core library of blocks?

Reply:
Core blocks. I guess `registerCoreBlocks()` is the way to go then. I had it typed once, but reverted to `all` keyword :)

Reply:
Updated.